doc/user/content/ingest-data/postgres/amazon-rds.md

```diff
@@ -57,18 +57,18 @@ As a first step, you need to make sure logical replication is enabled.
     - If logical replication is already on, skip to [Create a publication and a
       Materialize user section](#2-create-a-publication-and-a-replication-user).
 
-1. [Create a custom RDS parameter group](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithDBInstanceParamGroups.html#USER_WorkingWithParamGroups.Creating).
+1. [Create a db parameter group in RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithParamGroups.Creating.html) using the AWS management console..
 
     - Set **Parameter group family** to your PostgreSQL version.
     - Set **Type** to **DB Parameter Group**.
+    - Set **Engine type** to PostgreSQL
 
 1. Edit the new parameter group and set the `rds.logical_replication` parameter
    to `1`.
 
-1. [Associate the RDS parameter group to your database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithDBInstanceParamGroups.html#USER_WorkingWithParamGroups.Associating).
+1. [Associate the db parameter group with your database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithParamGroups.Associating.html).
 
-    Use the **Apply Immediately** option. The database must be rebooted in order
-    for the parameter group association to take effect. Keep in mind that
+    Use the **Apply Immediately** option to immediately reboot your database and apply the change.  Keep in mind that
```

Comment:
Added that there's a reboot so you users don't accidentally hurt their performance.